### ISSUE TITLE
How to get UI-popup?

### ISSUE THREAD

**Issue by vernikit** [gabrieljackson/mattermost-plugin-wrangler#192]
I cannot find a way to open UI-popup, which snapshot I see in the readme of the plugin. Option for webapp functionality is turned on in the settings. Also there are no additional buttons / signs. Why is that?

**Comment by vernikit** [comment]
@woorszt I finally found it: press cubes at the right corner of the message (”message actions”) and you will see the actions for Wrangler.

**Comment by woorszt** [comment]
Thanks @vernikit ! found it

**Comment by gabrieljackson** [comment]
> press cubes at the right corner of the message (”message actions”) and you will see the actions for Wrangler.

Yep, that's right. All plugin actions get registered there by default. I will try to update the README with a screenshot or two showing where to find this behavior.

Also, I will note that there is no wrangler functionality that is disabled for unlicensed Mattermost servers.
